src/cryptography/x509.py

```diff
@@ -1125,6 +1125,12 @@ def signature_hash_algorithm(self):
         in the certificate.
         """
 
+    @abc.abstractproperty
+    def extensions(self):
+        """
+        Returns a list of extensions
```

Comment:
That also works :-)

On Fri, May 15, 2015 at 12:32 PM, Paul Kehrer <notifications@github.com>
wrote:

> In src/cryptography/x509.py
> <https://github.com/pyca/cryptography/pull/1960#discussion_r30423803>:
>
> > @@ -1125,6 +1125,12 @@ def signature_hash_algorithm(self):
> >          in the certificate.
> >          """
> >
> > +    @abc.abstractproperty
> > +    def extensions(self):
> > +        """
> > +        Returns a list of extensions
>
> Should probably just say an Extensions object right? :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/1960/files#r30423803>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084